source/common/http/header_map_impl.cc

```diff
@@ -40,6 +40,12 @@ InlineHeaderVector& getInVec(VariantHeader& buffer) {
 const InlineHeaderVector& getInVec(const VariantHeader& buffer) {
   return absl::get<InlineHeaderVector>(buffer);
 }
+
+bool validatedLowerCaseString(absl::string_view str) {
+  auto lower_case_str = LowerCaseString(std::string(str));
```

Comment:
We might want to make LowerCaseString constructor takes string_view.

Reply:
get it.